mgmt_upgrade_test.py

```diff
@@ -51,9 +52,17 @@ def _create_and_add_cluster(self):
         return mgr_cluster, current_manager_version
 
     def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        manager_node = self.monitors.nodes[0]
+
         target_upgrade_server_version = self.params.get('target_scylla_mgmt_server_repo')
         target_upgrade_agent_version = self.params.get('target_scylla_mgmt_agent_repo')
```

Comment:
If this refer to url or repo file it should be clearer (and remove the upgrade from the name).

Reply:
target_manager_url  & target_manager_agent_url